products/extensions/b2b-components/employee-management/concept/api-route-restriction-for-employees.md

```diff
@@ -0,0 +1,31 @@
+# API Route Restriction for Employees
+
+## General
+
+B2B employees and Business Partners share the same customer account.
+This can lead to inconsistency for all users of the shared account because they are allowed to change settings and data (both via Storefront and Store-API),
+which are not related to the B2B permission system.
+We decided to restrict most of the customer account routes by implementing a deny-list pattern to prevent the illegal modification of customer data and settings,
+instead of replicating all customer features for employee accounts. We don't consider those features relevant to employee account users.
+
+## Deny-List
```

Comment:
```suggestion
## Denylist
```

I'll comment this once, since you can search-replace it. Reference: https://en.wikipedia.org/wiki/Blacklist_(computing)